spras/pathlinker.py

```diff
@@ -139,4 +139,5 @@ def parse_output(raw_pathway_file, standardized_pathway_file):
         # What about multiple raw_pathway_files
         df = pd.read_csv(raw_pathway_file, sep='\t').take([0, 1, 2], axis=1)
         df = reinsert_direction_col_directed(df)
-        df.to_csv(standardized_pathway_file, header=False, index=False, sep='\t')
+        df.columns = ['Node1', 'Node2', 'Rank', "Direction"]
+        df.to_csv(standardized_pathway_file, header=True, index=False, sep='\t')
```

Comment:
We want to better understand the behavior of `to_csv` when the data frame is empty. Our intended output is the header row instead of an empty file.

Reply:
I was testing what was going on with oi1. When we read in empty files, we catch the pd.errors.EmptyDataError error and don't write any df, which is most likely why no header is written. I checked same error on mcf which also showed that no df was being created when an empty file is read in. 

I changed the code to allow an empty df to be created and written out for oi1, but I want to double-check it before updating all of the other wrapper codes to do the same thing.

Reply:
I also just noticed that all the headers for the output pathway files aren't the same, so I will need to fix that too